archunit/src/main/java/com/tngtech/archunit/library/dependencies/SliceCycleArchCondition.java

```diff
@@ -212,13 +212,14 @@ private ConditionEvent newEvent(Cycle<Slice, Dependency> cycle) {
         private String createDescription(Collection<String> edgeDescriptions) {
             List<String> descriptions = new ArrayList<>(edgeDescriptions);
             descriptions.add(descriptions.get(0));
-            return Joiner.on(" -> ").join(descriptions);
+            return Joiner.on(" -> " + System.lineSeparator() + "                ").join(descriptions);
```

Comment:
I think it would be nicer to extract constants for the white space (e.g. `INDENT`) and also refer to other relevant constants, where necessary, so the relation would be immediately clear.
For example this long whitespace has to match the length of `"Cycle detected: "`, right? So I think it would be nicer to have something like `CYCLE_DESCRIPTION_INDENT = whitespace(CYCLE_DESCRIPTION.length())` or similar. What do you think?